### PR TITLE
Hover over broadcasts to see their owners

### PR DESCRIPTION
**Pull request by JamesHageman**
> Showing  14 changed files  with 404 additions and 31 deletions.

yeah boiiiiiiiiiiiiiiii

### CODE REVIEW

**Comment by georgeutsin** [comment]
Just some intial testing, im not sure if you git pull'd after the styles refactor, but a multiline broadcast makes things wiggle on hover

**Comment by georgeutsin** [comment]
Also, if you notice on the site, the "active hover area" is only the text, which doesn't include the sidebar on the right. Also a minor thing, there is a small strip non active hover area in between broadcasts (the margin)

**Comment by georgeutsin** [comment]
(basically move the onmousehover callback to the `broadcast-content-wrapper`

**Comment by JamesHageman** [comment]
@georgeutsin I can fix the measurement thing in this pr, I might need you to do the margin thing since I'm not as familiar with the design

File: src/actions.js [hunk 1]
```diff
@@ -40,7 +49,11 @@ export type Action =
   { type: 'BROADCASTS_FETCH', page: BroadcastPage } |
   { type: 'RECEIVE_BROADCASTS', broadcasts: Broadcast[], page: BroadcastPage } |
   { type: 'NEW_BROADCAST', text: string } |
-  { type: 'RECEIVE_NEW_BROADCAST', broadcast: Broadcast } 
+  { type: 'RECEIVE_NEW_BROADCAST', broadcast: Broadcast } |
+  { type: 'BROADCAST_DETAILS_SHOW', id: number } |
```
Comment (JamesHageman) [inline src/actions.js]: Ok, it should be BROADCAST_OWNER_{SHOW|HIDE}

Comment (georgeutsin) [inline src/actions.js]: Sounds good! That way namespace isn't polluted when we actually implement the show/hide of the details pane (where you can view the images, delete a broadcast, add a comment, flag it, etc)

**Comment by JamesHageman** [comment]
@georgeutsin I've implemented the measurement stuff, can you review again?

**Comment by JamesHageman** [comment]
I still need to update the avatar thing but maybe I'll do that in the next pr since this one is getting big

**Pull request merged**